SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKEditorSettings.cpp

```diff
@@ -109,32 +121,28 @@ void USpatialGDKEditorSettings::PostInitProperties()
 	PlayInSettings->PostEditChange();
 	PlayInSettings->SaveConfig();
 
-	SetRuntimeWorkerTypes();
 	SetRuntimeUseDevelopmentAuthenticationFlow();
 	SetRuntimeDevelopmentAuthenticationToken();
 	SetRuntimeDevelopmentDeploymentToConnect();
-}
 
-void USpatialGDKEditorSettings::SetRuntimeWorkerTypes()
-{
-	TSet<FName> WorkerTypes;
-	
-	for (const FWorkerTypeLaunchSection& WorkerLaunch : LaunchConfigDesc.ServerWorkers)
+	const USpatialGDKSettings* GDKSettings = GetDefault<USpatialGDKSettings>();
+
+	if (LaunchConfigDesc.ServerWorkers_DEPRECATED.Num() > 0)
```

Comment:
How come this is called `ServerWorkers_DEPRECATED`?

Reply:
This allows to load the previous parameters, while signaling that this property is only there for compatibility purposes, and not save it again.